modules/services/avizo.nix

```diff
@@ -0,0 +1,74 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let cfg = config.services.avizo;
+in {
+
+  meta.maintainers = [ hm.maintainers.pltanton ];
+
+  options.services.avizo = {
+    enable = mkOption {
+      default = false;
+      description = ''
+        Whether to enable avizo.
+        Avizo is a simple notification daemon, mainly intended to be used for multimedia keys for example with Sway.
+      '';
+    };
+    config = mkOption {
+      type = types.attrs;
+      default = { };
+      example = {
+        default = {
+          time = 1.0;
+          y-offset = 0.5;
+          fade-in = 0.1;
+          fade-out = 0.2;
+          padding = 10;
+        };
+      };
+      description =
+        "Attrs that will be converted to .ini avizo configuration file.";
+    };
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.easyeffects;
+      defaultText = literalExpression "pkgs.avizo";
+      example = literalExpression ''
+        pkgs.avizo.overrideAttrs (final: prev: {
+          patchPhase = "cp ''${./images}/*.png data/images/
+        });
```

Comment:
This is not a valid Nix expression (unfinished line, final semicolon)

Reply:
Fixed